handout/testing/services/alt-http-mocking.md

```diff
@@ -0,0 +1,83 @@
+# Alternative HTTP Mocking Strategy
+
+An alternative to using `MockBackend` is to create light mocks by ourselves. Here we create an object and then tell TypeScript to treat it as `Http` using type assertion. We then create a spy for its `get` method and return an observable similar to what the real `Http` service would do.
```

Comment:
How about:

An alternative to using `MockBackend` is to create our own light mocks ~~by ourselves~~.